README.md

```diff
@@ -14,14 +14,14 @@ This repository holds the source files for building a minimal docker image of th
 *   [Docker](https://www.docker.com): `18-stable`
 
 ## How to use
-Pre-build images are available on [DockerHub](https://hub.docker.com/r/duncdrum/exist-docker/). There are two channels:
-*   `stable` for the latest stable releases (coming soon™)
+Pre-build images are available on [DockerHub](https://hub.docker.com/r/existdb/existdb/). There are two channels:
+*   `release` for the latest stable releases based on the master branch
```

Comment:
Can we say `master` branch, and link to it

Reply:
no latest and release are the common terminology, our ability to directly interact with branches in the main repo is limited, since we keep the docker stuff in a separate repo

Reply:
Oh i think i misunderstood, see latest commit